app/listeners/webhook_listener.rb

```diff
@@ -51,6 +51,21 @@ def webwidget_triggered(event)
     deliver_webhook_payloads(payload, inbox)
   end
 
+  def contact_created(event)
+    contact = extract_contact_and_account(event)[0]
+    inbox = conversation.inbox
+    payload = contact.webhook_data.merge(event: __method__.to_s)
+    deliver_webhook_payloads(payload, inbox)
```

Comment:
Since the inbox object is not associated with the contact, we might need to update the method deliver_account_webhooks as shown below.

```rb
  def contact_created(event)
      # ...
      deliver_webhook_payloads(payload, contact.account)
  end

  # ..

  def deliver_account_webhooks(payload, account)
    account.webhooks.account_type.each do |webhook|
      next unless webhook.subscriptions.include?(payload[:event])

      WebhookJob.perform_later(webhook.url, payload)
    end
  end
```

Reply:
Thanks